src/platform/meteorlake/include/platform/lib/clk.h

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *         Keyon Jie <yang.jie@linux.intel.com>
+ *         Rander Wang <rander.wang@intel.com>
+ */
+
+#ifdef __SOF_LIB_CLK_H__
+
+#ifndef __PLATFORM_LIB_CLK_H__
+#define __PLATFORM_LIB_CLK_H__
+
+#include <ace/lib/clk.h>
+
+#define CLK_MAX_CPU_HZ		400000000
+
+#define CPU_WOVCRO_FREQ_IDX	0
+
+#define CPU_LPRO_FREQ_IDX	1
+
+#define CPU_HPRO_FREQ_IDX	2
```

Comment:
These shouldn't be needed either as clocks are managed on Zephyr side.

Reply:
Similar to the above